### PR TITLE
P3 manage cache

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds information about managing the P3 cache in the UI. Reviewers, please refer to my questions in the comments.

### CODE REVIEW

File: docs/photoniq/p3/manage-p3-cache.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Perform the following steps to purge your P3 cache:
+
+1. Log in to your PhotonIQ Performance Proxy dashboard.
+2. Click **Cache**.
+3. Click **Purge Cache**.
+4. Click **Add** and then enter the URL, including `https://`, that you want to purge the cache for. Repeat this step for each URL you want to include in the purge request.
+5. (Optional) If you want to purge your CDN cache as well, then click the **Yes, I want to purge CDN cache too.** checkbox.
+6. If purging the CDN, then click the arrow to expand the **CDN Purge Params** and then enter information in the following fields:
+
+   - Network
```
Comment (diana-macrometa) [inline docs/photoniq/p3/manage-p3-cache.md]: We need to either provide definitions for all these fields or link to an Akamai page where they are defined. Also, do we want to say that Akamai is the only CDN we support?

File: docs/photoniq/p3/manage-p3-cache.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## View Purge Cache Requests
+
+Perform the following steps to view P3 cache purge requests:
+
+1. Log in to your PhotonIQ Performance Proxy dashboard.
+2. Click **Cache**.
+
+    The Purge Cache Requests table provides the following fields:
+
+    - **URL** - The URL of the cache being purged.
```
Comment (diana-macrometa) [inline docs/photoniq/p3/manage-p3-cache.md:19]: I need some details here. What are the requirements for the URL? Origin URL? Part of the Origin? What does the user need to know about this?

Comment (dwivedianurag) [reply docs/photoniq/p3/manage-p3-cache.md:19]: The user needs to know the p3 origin url and their url pattern or url for which he has configured the P3 to work on .

File: docs/photoniq/prerendering/manage-cache/manage-cache.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ You have several options for managing the Prerendering cache.
 
 ## Cache Manager Fields
 
-Prerendering provides the cache manager fields. Field names in the web console are different than those in the API responses. Refer to the API endpoint documentation for a full explanation of the API response.
+Prerendering provides the following cache manager fields. Field names in the web console are different than those in the API responses. Refer to the API endpoint documentation for a full explanation of the API response.
```
Comment (dwivedianurag) [inline docs/photoniq/prerendering/manage-cache/manage-cache.md:16]: Prerendering it should be P3

**Review by dwivedianurag — APPROVED** [review]
ok

**Comment by dwivedianurag** [comment]
https://techdocs.akamai.com/purge-cache/docs/welcome-purge

File: docs/photoniq/p3/manage-p3-cache.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/photoniq/p3/manage-p3-cache.md]: @durgagokina, can you answer this question?

**Pull request merged**